lib/galaxy/tools/imp_exp/unpack_tar_gz_archive.py

```diff
@@ -64,7 +64,7 @@ def unpack_archive(archive_file, dest_dir):
     """
     Unpack a tar and/or gzipped archive into a destination directory.
     """
-    archive_fp = tarfile.open(archive_file, mode='r')
+    archive_fp = tarfile.open(archive_file, mode='rb')
```

Comment:
`rb` is not listed among the allowed modes in https://docs.python.org/3/library/tarfile.html#tarfile.open , is that necessary?

Reply:
Thanks, it is not, there were and are still other issues with the import/export , some of which I fixed in https://github.com/galaxyproject/galaxy/pull/6561/commits/025167546202591524a41ae1f8ceddb859cbac37